File: tests/struct/in-tuple.js
export function makeSizes$w1$h1$w2$h2$(w1, h1, w2, h2) {
  return [{
    width: w1,
    height: h1
  }, {
    width: w2,
    height: h2
  }];
}
export function sumSizes$sizes$(sizes) {
  let w = sizes[0].width + sizes[1].height;
  let h = sizes[0].height + sizes[1].height;
  return {
    width: w,
    height: h
  };
}
export function copySizes$sizes$(sizes) {
  return [{
    width: sizes[0].width,
    height: sizes[0].height
  }, {
    width: sizes[1].width,
    height: sizes[1].height
  }];
}

File: tests/struct/in-tuple.swift
public struct Size {
    var width: Double
    var height: Double
    public init(width w: Double, height h: Double) {
        width = w
        height = h
    }
}

public func makeSizes(w1: Double, h1: Double, w2: Double, h2: Double) -> (Size, Size) {
    return (Size(width: w1, height: h1), Size(width: w2, height: h2))
}

public func sumSizes(sizes: (Size, Size)) -> Size {
	return Size(width: sizes.0.width + sizes.1.height, height: sizes.0.height + sizes.1.height)
}

public func copySizes(sizes: (Size, Size)) -> (Size, Size) {
	return sizes
}
